recipes/libsndfile/all/patches/0001-turn-to-warning.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/cmake/SndFileChecks.cmake b/cmake/SndFileChecks.cmake
+index 3528c325..b15e3ab4 100644
+--- a/cmake/SndFileChecks.cmake
++++ b/cmake/SndFileChecks.cmake
+@@ -213,26 +213,7 @@ if (MSVC)
+ 	add_definitions (-D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE)
+ endif (MSVC)
+ 
+-if (MINGW)
```

Comment:
you could also use `if (FALSE)` instead of `if (MINGW)` to decrease patch size, but that's fine.

Reply:
Done